public/js/app/components/Table.jsx

```diff
@@ -1,28 +1,33 @@
 import React from 'react'
 
-const renderRow = (row) => (
-  <tr key={row[0]}>
-    {row.map((cell, index) => (
-      <td key={index}>{typeof cell === 'function' ? cell() : cell}</td>
-    ))}
-  </tr>
-)
+const renderRow = (row, rowIndex) => {
+  return (
+    //TODO: unique ID for key
+    <tr key={rowIndex}>
+      {row.map((cell, cellIndex) => (
+        <td key={cellIndex}>{typeof cell === 'function' ? cell() : cell}</td>
+      ))}
+    </tr>
+  )
+}
 
-const Table = ({ headings, rows, tableClasses }) => (
-  <>
-    <table className={tableClasses.join(' ')}>
-      <thead>
-        <tr>
-          {headings.map((h) => (
-            <th key={h} scope="col">
-              {h}
-            </th>
-          ))}
-        </tr>
-      </thead>
-      <tbody>{rows.map((r) => renderRow(r))}</tbody>
-    </table>
-  </>
-)
+const Table = ({ headings, rows, tableClasses }) => {
+  return (
+    <>
+      <table className={tableClasses.join(' ')}>
+        <thead>
+          <tr>
+            {headings.map((heading, index) => (
```

Comment:
I had to reread the section about keys in the react-docs. https://react.dev/learn/rendering-lists#rules-of-keys
This leaves me with the understanding that we should not use indices or something else that changes at render. But it is enough if the key is unique _within the same array_, we can use the same key in different arrays.

With this in mind these are my thoughts:
- the headings here are never supposed to change - is this actually a situation where index as key would be working?
- we should be fine using the `heading` itself as a key, because we know that it is unique.